client/galaxy/scripts/mvc/toolshed/repositories-view.js

```diff
@@ -114,45 +123,45 @@ var ToolShedCategoryContentsView = Backbone.View.extend({
             "<% } else { %>",
             '<a data-page="1" class="pagenav-inactive fa fa-fast-backward" />',
             '<a data-page="<%= previous %>" class="pagenav-inactive fa fa-step-backward" />',
-            '<% } %>',
-            '<% if (pages > 5) { %>',
-            '<% if (page != 1) { %>',
+            "<% } %>",
+            "<% if (pages > 5) { %>",
+            "<% if (page != 1) { %>",
             '<a data-page="1" class="pagenav fa"><a>1</a>',
-            '<% if (page != 2) { %>',
+            "<% if (page != 2) { %>",
             '<a class="fa">&hellip;</a>',
-            '<% } %>',
-            '<% } else { %>',
+            "<% } %>",
+            "<% } else { %>",
             '<a data-page="1" class="pagenav-inactive fa"><a>1</a>',
-            '<% } %>',
-            '<% _.each(page_slice, function(i) { %>',
-            '<% if (i == page) { %>',
+            "<% } %>",
```

Comment:
lol, why prettier, why?!

Reply:
Consistency in quoting!  :)

Reply:
nope, it leaves the above line with `'`, if anything this decreases consistency

Reply:
Ahh, I see what you mean.  Yeah, if there's a conflict it won't mangle the contents of your string obviously.

Reply:
right...the consistency is everpresent in this approach 😭

Reply:
Multiple layers of consistency!  At least when everyone runs prettier it comes out the same :)